source/Playnite/Controllers/GenericGameController.cs

```diff
@@ -20,26 +21,43 @@ public class GenericGameController : BaseGameController
         protected Stopwatch stopWatch;
         protected ProcessMonitor procMon;
         private GameDatabase database;
+        private static ILogger logger = LogManager.GetLogger();
 
-        public GenericGameController(GameDatabase db, Game game) : base (game)
+        public GenericGameController(GameDatabase db, Game game) : base(game)
         {
             database = db;
         }
 
         public override void Play()
-        {            
+        {
             if (Game.PlayAction == null)
             {
                 throw new Exception("Cannot start game without play action.");
             }
 
-            var playAction = Game.PlayAction.ExpandVariables(Game);
+            var gameClone = Game.GetClone();
+            var playAction = gameClone.PlayAction;
+            var emulators = database.Emulators.ToList();
+            var profileClone = GameActionActivator.GetGameActionEmulatorConfig(playAction, emulators)?.GetClone();
+
+            //Solving Issue #1065 if removable drives are used for storing ROMs/Images

```

Comment:
Can you remove references to #1065? I don't think it's necessary. This is not workaround for some issue, but new feature and I usually don't add references for those.